File: 코딩테스트/그리디/1541.py
arr = input().split("-")
s=0
result=[]
for i in arr:
    if len(i.split("+")) > 1:
        k=0
        for j in i.split("+"):
           k+=int(j)
        result.append(k)
    else:
        result.append(int(i))
s+=result[0]
for i in range(1,len(result)):
    s-=result[i]
print(s)

File: BAEKJOON/greedy/1541.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const arr = input[0].split("-");
let result = 0;
for (let i = 0; i < arr.length; i++) {
  const number = arr[i]
    .split("+")
    .map(Number)
    .reduce((acc, cur) => acc + cur);
  if (i == 0) result += number;
  else result -= number;
}
console.log(result);
